src/mcp_agent/mcp/mcp_aggregator.py

```
@@ -241,108 +272,156 @@
-        results = await asyncio.gather(
-            *(self.load_server(server_name) for server_name in self.server_names),
-            return_exceptions=True,
-        )
-
-        for result in results:
-            if isinstance(result, BaseException):
-                logger.error(
-                    f"Error loading server data: {result}. Attempting to continue"
-                )
-                continue
+            for server_name, result in zip(self.server_names, results):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)